src/Feed/EventDatabaseApiFeedType.php

```diff
@@ -19,107 +21,116 @@ class EventDatabaseApiFeedType implements FeedTypeInterface
+
+                        return $decoded->{'hydra:member'};
+                    case 'single':
+                        if (isset($configuration['singleSelectedOccurrence'])) {
+                            $occurrenceId = $configuration['singleSelectedOccurrence'];
+
+                            $response = $this->client->request(
+                                'GET',
+                                "$host$occurrenceId",
+                                [
+                                    'timeout' => self::REQUEST_TIMEOUT,
+                                ]
+                            );
+
+                            $content = $response->getContent();
+                            $decoded = json_decode($content);
+
+                            $baseUrl = parse_url($decoded->event->{'url'}, PHP_URL_HOST);
+
+                            $eventOccurrence = (object) [
+                                'eventId' => $decoded->event->{'@id'},
+                                'occurrenceId' => $decoded->{'@id'},
+                                'ticketPurchaseUrl' => $decoded->event->{'ticketPurchaseUrl'},
+                                'excerpt' => $decoded->event->{'excerpt'},
+                                'name' => $decoded->event->{'name'},
+                                'url' => $decoded->event->{'url'},
+                                'baseUrl' => $baseUrl,
+                                'image' => $decoded->event->{'image'},
+                                'startDate' => $decoded->{'startDate'},
+                                'endDate' => $decoded->{'endDate'},
+                                'ticketPriceRange' => $decoded->{'ticketPriceRange'},
+                                'eventStatusText' => $decoded->{'eventStatusText'},
                             ];
-                        }
 
-                        return [$eventOccurrence];
-                    }
+                            if (isset($decoded->place)) {
+                                $eventOccurrence->place = (object) [
+                                    'name' => $decoded->place->name,
+                                    'streetAddress' => $decoded->place->streetAddress,
+                                    'addressLocality' => $decoded->place->addressLocality,
+                                    'postalCode' => $decoded->place->postalCode,
+                                    'image' => $decoded->place->image,
+                                    'telephone' => $decoded->place->telephone,
+                                ];
+                            }
+
+                            return [$eventOccurrence];
+                        }
+                }
+            }
+        } catch (\Throwable $throwable) {
+            // If the content does not exist anymore, unpublished the slide.
+            if ($throwable instanceof ClientException && 404 == $throwable->getCode()) {
```

Comment:
```suggestion
            if ($throwable instanceof ClientException && Response::HTTP_NOT_FOUND == $throwable->getCode()) {
```

(cf. https://github.com/symfony/symfony/blob/6.3/src/Symfony/Component/HttpFoundation/Response.php#L51)